recipes/cairo/all/conanfile.py

```diff
@@ -289,8 +280,6 @@ def package_info(self):
             if not self.options.shared:
                 self.cpp_info.components["cairo_"].defines.append("CAIRO_WIN32_STATIC_BUILD=1")
         else:
-            if self.options.with_glib:
-                self.cpp_info.components["cairo_"].requires.extend(["glib::gobject-2.0", "glib::glib-2.0"])
```

Comment:
Any insight into these changes?

Reply:
I'd say it was an error from the start https://github.com/conan-io/conan-center-index/pull/2847/commits/8619fc3493754a1870bac3352d46d6723acff30a#diff-5fbc6d75ac2a534cc5331a7d03445181fd9361da70cbbe9fb23873f06f5cc023R223

The main lib never linked to glib or gobject https://gitlab.freedesktop.org/cairo/cairo/-/blob/1.17.2/src/Makefile.am?ref_type=tags
Only cairo-gobject does https://gitlab.freedesktop.org/cairo/cairo/-/blob/1.17.2/util/cairo-gobject/Makefile.am?ref_type=tags (as can be seen in the newer meson recipe https://github.com/conan-io/conan-center-index/blob/master/recipes/cairo/meson/conanfile.py#L233